java/src/jmri/jmrix/loconet/LocoNetThrottle.java

```diff
@@ -303,6 +303,86 @@ protected void sendFunctionGroup5() {
         adapterMemo.get(jmri.CommandStation.class).sendPacket(result, 4); // repeat = 4
     }
 
+    /**
+     * Sends Function Group 6 values - F29 thru F36, using an "OPC_IMM_PACKET" LocoNet
+     * Message.
+     */
+    @Override
+    protected void sendFunctionGroup6() {
+        // LocoNet practice is to send as a DCC packet
+        int i = 29;
+        byte[] result = jmri.NmraPacket.function29Through36Packet(address, (address >= 128),
+                getFunction(i), getFunction(i+1), getFunction(i+2), getFunction(i+3),
+                getFunction(i+4), getFunction(i+5), getFunction(i+6), getFunction(i+7));
+
+        log.debug("sendFunctionGroup5 sending {} to LocoNet slot {}", result, slot.getSlot());
```

Comment:
Copy/paste error:
`sendFunctionGroup5 sending {} to LocoNet slot {}`
The 5 should be 6.

Note that this applies to sendFunctionGroup7() to sendFunctionGroup10() as well.

Reply:
Fixed, thanks for catching that.